app/assets/v2/js/user-search.js

```diff
@@ -29,6 +29,14 @@ $(function() {
     templateSelection: formatUserSelection
   });
 
+  // fix for wrong position on select open
+  var select2Instance = $('.username-search').data('select2');
```

Comment:
@octavioamu correct me if I am wrong. But won't this return the value in the `data-select2` attribute instead of returning the select2 instance? Shouldn't we be doing something like `$('.username-search').find('data-select2')` ?

Reply:
Because the plugin need the data to recalculate the size of the box.

Reply:
In others words is the way to access the plugin functions and data not just the dom element

Reply:
Okay. 👍